polaris-service/src/main/java/org/apache/polaris/service/PolarisApplication.java

```diff
@@ -261,10 +261,7 @@ public void run(PolarisApplicationConfig configuration, Environment environment)
         .addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, "/*");
 
     if (configuration.getRateLimiter() != null) {
-      environment
-          .servlets()
-          .addFilter("ratelimiter", new RateLimiterFilter(configuration.getRateLimiter()))
-          .addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST), true, "/*");
+      environment.jersey().register(new RateLimiterFilter(configuration.getRateLimiter()));
```

Comment:
This change is unrelated to metrics, right ?

Reply:
See https://github.com/apache/polaris/pull/423#discussion_r1828923405